apps/web/src/components/conditions/Conditions.tsx

```diff
@@ -26,6 +26,19 @@ import { OnlineConditionRow } from './OnlineConditionRow';
 import { DefaultGroupOperatorData, DefaultOperatorData } from './constants';
 import { PreviousStepsConditionRow } from './PreviousStepsConditionRow';
 
+export interface IConditionsComponentProps {
+  isOpened: boolean;
+  isReadonly?: boolean;
+  onClose: () => void;
+  updateConditions: (data: IConditions[]) => void;
+  conditions?: IConditions[];
+  name: string;
+  label?: string;
+  filterPartsList: IFilterTypeList[];
+  defaultFilter?: FilterPartTypeEnum;
+  shouldDisallowEmptyConditions?: boolean;
```

Comment:
Flag here is optional / opt-in to preserve the original functionality for other callers of this component.